openassessment/assessment/test/test_staff.py

```diff
@@ -0,0 +1,348 @@
+
+STUDENT_ITEM = dict(
+    student_id="Tim",
+    course_id="Demo_Course",
+    item_id="item_one",
+    item_type="Peer_Submission",
+)
+
+ANSWER_ONE = u"this is my answer!"
+
+# Possible points: 14
+RUBRIC_DICT = {
+    "criteria": [
+        {
+            "name": "secret",
+            "prompt": "Did the writer keep it secret?",
+            "options": [
+                {"name": "no", "points": "0", "explanation": ""},
+                {"name": "yes", "points": "1", "explanation": ""},
+            ]
+        },
+        {
+            "name": u"ⓢⓐⓕⓔ",
+            "prompt": "Did the writer keep it safe?",
+            "options": [
+                {"name": "no", "points": "0", "explanation": ""},
+                {"name": "yes", "points": "1", "explanation": ""},
+            ]
+        },
+        {
+            "name": "giveup",
+            "prompt": "How willing is the writer to give up the ring?",
+            "options": [
+                {
+                    "name": "unwilling",
+                    "points": "0",
+                    "explanation": "Likely to use force to keep it."
+                },
+                {
+                    "name": "reluctant",
+                    "points": "3",
+                    "explanation": "May argue, but will give it up voluntarily."
+                },
+                {
+                    "name": "eager",
+                    "points": "10",
+                    "explanation": "Happy to give it up."
+                }
+            ]
+        },
+        {
+            "name": "singing",
+            "prompt": "Did the writer break into tedious elvish lyrics?",
+            "options": [
+                {"name": "no", "points": "2", "explanation": ""},
+                {"name": "yes", "points": "0", "explanation": ""}
+            ]
+        },
+    ]
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 6 points
+ASSESSMENT_DICT = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {
+        "giveup": u"𝓨𝓸𝓾 𝓼𝓱𝓸𝓾𝓵𝓭𝓷'𝓽 𝓰𝓲𝓿𝓮 𝓾𝓹!"
+    },
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "reluctant",
+        "singing": "no",
+    },
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 0 points
+ASSESSMENT_DICT_FAIL = {
+    'overall_feedback': u"fail",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "no",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "unwilling",
+        "singing": "yes",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+ASSESSMENT_DICT_PASS = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "yes",
+        "giveup": "eager",
+        "singing": "no",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+# Feedback text is one character over the limit.
+LONG_FEEDBACK_TEXT = u"是" * Assessment.MAX_FEEDBACK_SIZE + "."
```

Comment:
This, and several other things in this file seem to be unneeded remnants of what I copy-pasted over to get things started. I'll create a new PR to remove them.